packages/dcos-integration-test/extra/test_units.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+
+import pytest
+
+
+def test_verify_units():
+    """Test that all systemd units are valid."""
```

Comment:
@gpaul can you confirm that this catches the broken LimitNOFILE= directive? We were thinking that what we really want (maybe in addition to this) is checks on /proc/<pid>/limits. This is admittedly a bit tricky, since e.g. Spartan ends up with 3 processes running in its unit (maybe we can fix that too).